src/lib.rs

```diff
@@ -150,8 +156,30 @@ impl<T: Runtime> ClientBuilder<T> {
         self
     }
 
+    /// Register a custom type segmenter, for consuming types in events where the size cannot
+    /// be inferred from the metadata.
+    ///
+    /// # Panics
+    ///
+    /// If there is already a type size registered with this name.
+    pub fn register_type_size<U>(mut self, name: &str) -> Self
+    where
+        U: Codec + Send + Sync + 'static,
+    {
+        self.event_type_registry.register_type_size::<U>(name);
+        self
+    }
+
+    /// Disable the check for missing type sizes on `build`.
+    ///
+    /// *WARNING* can lead to runtime errors if receiving events with unknown types.
+    pub fn skip_type_sizes_check(mut self) -> Self {
+        self.skip_type_sizes_check = false;
```

Comment:
is this should be `self.skip_type_sizes_check = true;` ?

Reply:
Indeed looks like it should be...

Reply:
https://github.com/paritytech/substrate-subxt/pull/235